### PR TITLE
Soft break inside list element will be handled correctly.

### PR DESCRIPTION
**Pull request by pomek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Fix: Block filler will be inserted into the list item if its last child is a `<br>` element. Closes ckeditor/ckeditor5#1312.

### CODE REVIEW

**Review by Reinmar — CHANGES_REQUESTED** [review]
Couldn't we reuse the logic of the original `getFillerOffset()`?  Isn't this function just an extension of the original one?

**Comment by pomek** [comment]
Done. This PR requires https://github.com/ckeditor/ckeditor5-engine/pull/1591 now.

image

File: src/utils.js [hunk 1]
```diff
@@ -15,16 +17,20 @@
...
+	viewItem.getFillerOffset = _getFillerOffset;
 
 	return viewItem;
 }
 
 // Implementation of getFillerOffset for view list item element.
 //
 // @returns {Number|null} Block filler offset or `null` if block filler is not needed.
-function getFillerOffset() {
+function _getFillerOffset() {
```
Comment (Reinmar) [inline src/utils.js]:
```suggestion
function getListItemFillerOffset() {
```

You can be more explicit here and don't force people to understand that there are two functions available here in the scope.

File: src/utils.js [hunk 2]
```diff
@@ -15,16 +17,20 @@
  */
 export function createViewListItemElement( writer ) {
 	const viewItem = writer.createContainerElement( 'li' );
-	viewItem.getFillerOffset = getFillerOffset;
+	viewItem.getFillerOffset = _getFillerOffset;
```
Comment (pomek) [inline src/utils.js]:
```suggestion
	viewItem.getFillerOffset = getListItemFillerOffset;
```

**Review by Reinmar — APPROVED** [review]

**Pull request merged**